ras/sosreport.py

```diff
@@ -57,14 +57,14 @@ def setUp(self):
             self.sos_cmd = "sosreport"
         elif dist.name in ['rhel', 'centos', 'fedora']:
             sos_pkg = 'sos'
+            self.sos_cmd = "sos report"
```

Comment:
older distro still has command `sosreport`

Reply:
Logic for older distro version, i.e., for RHEL7.4 and below has been handled.
Thanks.